docs/DeveloperGuide.adoc

```diff
@@ -806,24 +808,25 @@ Use case ends.
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 2a. The list is empty
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The given index is invalid
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. AddressBook shows an error message
 +
 Use case resumes at step 2.
 
 [discrete]
-=== Use case: Create a contact
+=== Use case: Add a person
 
 *MSS*
 
-1. User inputs the command with the correct keywords
-2. AddressBook confirms the addition by showing the confirmation message
+1. User requests to add person
+2. User inputs person details
```

Comment:
In the case of MSS, it should be a one to one interaction between user and system:
User does something
System responds
User does something
System responds.
In this case `line 1 and 2` can be merged together to reflect this behaviour.